renovate.json

```diff
@@ -1,47 +1,17 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
   "extends": [
-    "config:base",
-    ":rebaseStalePrs",
-    "docker:disableMajor"
+    "github>gsoft-inc/renovate-config"
   ],
-  "enabledManagers": [
-    "nuget",
-    "dockerfile",
-    "github-actions"
-  ],
-  "stabilityDays": 3,
-  "prHourlyLimit": 0,
-  "prConcurrentLimit": 0,
-  "branchConcurrentLimit": 0,
-  "dependencyDashboard": false,
-  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
-  "nuget": {
-    "extends": [":disableMajorUpdates"]
-  },
   "packageRules": [
     {
-      "matchManagers": ["nuget"],
-      "groupName": "NuGet dependencies"
-    },    
-    {
-      "matchManagers": ["nuget"],
-      "matchPackageNames": ["NuGet Azure.Messaging.EventGrid"],
+      "matchManagers": [
+        "nuget"
+      ],
+      "matchPackageNames": [
+        "NuGet Azure.Messaging.EventGrid"
```

Comment:
I think we don't need this one. There is now a stable version of the package (see https://github.com/gsoft-inc/wl-domain-event-propagation/pull/159)